lib/ohai/plugins/linux/platform.rb

```diff
@@ -59,6 +59,15 @@ def get_redhatish_version(contents)
       contents = File.read("/etc/parallels-release").chomp
       platform get_redhatish_platform(contents)
       platform_version contents.match(/(\d\.\d\.\d)/)[0]
+    elsif File.exists?('/etc/os-release')
```

Comment:
This is the new standard for this release information in the future, so you will break the world if you put this first and don't have detection for everyone else in here.